### DISCUSSION TITLE
提示需要下载一个exe文件，但是目前抢票不影响使用

### DISCUSSION THREAD

**Discussion by 1321312424124** [mikumifa/biliTickerBuy#155] [提问专区] [answered]
<img width="662" alt="屏幕截图 2024-06-22 221336" src="https://github.com/mikumifa/biliTickerBuy/assets/171526978/8d86a0f0-9bc6-4273-b97f-895fe1b08005">
这个会影响使用吗？

**Comment by WittF** [comment, marked as answer]
参考 https://github.com/mikumifa/biliTickerBuy/discussions/120
